Source/Scene/GoogleEarthEnterpriseMapsProvider.js

```diff
@@ -91,14 +94,14 @@ define([
      *
      *
      * @example
-     * var google = new Cesium.GoogleEarthImageryProvider({
+     * var google = new Cesium.GoogleEarthEnterpriseMapsProvider({
      *     url : 'https://earth.localdomain',
      *     channel : 1008
      * });
      *
      * @see {@link http://www.w3.org/TR/cors/|Cross-Origin Resource Sharing}
      */
-    function GoogleEarthImageryProvider(options) {
+    function GoogleEarthEnterpriseMapsProvider(options) {
```

Comment:
Since renaming this is a breaking change, we need to follow the [deprecation procedure](https://github.com/AnalyticalGraphicsInc/cesium/blob/6394242f4b96816f57707277e9b4bf42ccec9f3d/Documentation/Contributors/CodingGuide/README.md#deprecation-and-breaking-changes)

We'll have to keep both files around for now and mark the `GoogleEarthImageryProvider` as deprecated.  Then if one of our users is still using this imagery provider, we can use a `deprecationWarning` to print out to the console that the class they are using is deprecated, and that they can fix it by using the renamed class instead.  I would set this to be deprecated 3 releases from now, so in 1.37.